_content/papers/vimuttayatana_analayo.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Vimuttāyatana"
+authors:
+ - analayo
+editor: "W. G. Weeraratne"
+booktitle: "Encyclopaedia of Buddhism"
+volume: 8
+number: 3
+pages: "613--615"
+publisher: "The Department of Buddhist Affairs"
+address: "Sri Lanka"
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/encyclopedia-entries/vimuttayatana.pdf"
+year: 2009
+drive_links:
+ - "https://drive.google.com/file/d/1foJ10YJ8C0-kwAbKxYxF5apzMr9zu787/view?usp=sharing"
+course: sati
+tags:
+  - factors
```

Comment:
`factors` is a bit vague... You already have samādhi and vimutti, so let's just remove this one.